client/src/containers/pages/locations/Location.css

```diff
@@ -0,0 +1,32 @@
+.table-container {
+  background-color: #ffffff;
+  margin: 20px;
+}
+
+.location-table-action {
+  display: flex;
+  justify-content: space-around;
+  width: 86px;
+  align-items: center;
+}
+
+.location-table-action .edit {
+  margin: 0px;
+  cursor: pointer;
+  width: 45px;
+  text-align: center;
+  color: #1890ff;
+}
+
+.location-table-action .more-options {
+  cursor: pointer;
+  width: 32px;
+  color: #1890ff;
+  border-left: 1px solid;
+  height: 16px;
+  font-size: 18px;
+  border-left-color: #e9e9e9;
+}
+table tr:nth-child(2n) td {
+  background-color: #f4f4f4;
+}
```

Comment:
@HaseemUlHaq please remove custom css and try to reusable the css provided with bootstrap and antd rather than writing each style yourself

Reply:
Added this css for Stripped row design for Ant tables.
They are not providing any default class for stripped rows.